src/parameters.rs

```diff
@@ -43,12 +43,12 @@ pub(crate) const ELL_PRIME: usize = 5 * SECURITY_PARAM;
 
 /// The flex space of a range check.
 ///
-/// When a prover has a secret input `x` in the range `+/- 2^l`, the range check proofs
-/// guarantee to the verifier that `x` is in the range `+/- 2^(l + EPSILON)`.
+/// When a prover has a secret input `x` in the range `+/- 2^l`, the verifier can check that the
+/// masked proof response corresponding to `x` is in the range `+/- 2^(l + EPSILON)`.
```

Comment:
note: now that I understand the proof, I realized that these docs were not really correct.